normits_demand/models/notem/production_models.py

```diff
@@ -329,26 +323,21 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
+                self._logger.info("No code implemented to constrain productions." )
                 raise NotImplemented(
                     "No code implemented to constrain productions."
                 )
 
             # Print timing stats for the year
             year_end_time = timing.current_milli_time()
             time_taken = timing.time_taken(year_start_time, year_end_time)
-            du.print_w_toggle(
-                "HB Productions in year %s took: %s\n" % (year, time_taken),
-                verbose=verbose
-            )
+            self._logger.info("HB Productions in year " + year+"took :"+time_taken)
```

Comment:
Please keep the `%s` formatting that was in the original code.
This method of joining strings goes against the coding standards of this codebase

Reply:
Updated